research.md

```diff
@@ -39,6 +39,9 @@ Lots more hacks built using our API can be found at the [Perspective Hacks Galle
  
 ## Research Contributions
 
+* [Toxicity Detection: Does Context Really Matter?
+](https://www.aclweb.org/anthology/2020.acl-main.396/) studies the importance of  context, and when as well as to what extent it can change the perceived toxicity of posts, finding that context can both amplify or mitigate the perceived toxicity of posts, and moreover for a subset of the data that including context can even reverse the toxicity label.
```

Comment:
weird new line inside [].

Reply:
Weird, not sure how that happened. Doesn't appear to affect rendering, but fixed it.